src/components/tx-flow/flows/AddOwner/ChooseOwner.tsx

```diff
@@ -30,12 +30,19 @@ import { TOOLTIP_TITLES } from '@/components/tx-flow/common/constants'
 
 type FormData = Pick<AddOwnerFlowProps | ReplaceOwnerFlowProps, 'newOwner' | 'threshold'>
 
+export enum ChooseOwnerMode {
```

Comment:
We could remove this and just check if "removeOwner" is defined in the form params. 
I preferred this solution as its more readable / understandable IMO